blockchain/blockchain-wallet-v4-frontend
# Feature/payment requests

## Description
403, 726
Correctly label BTC address when payment request is 'created'.

## Change Type
Please enter one or more of the following: 
- Feature

## Testing Steps
Create a payment request, see address is labeled as unused address, receive payment and see label is seen in tx desc

## PR Creator Checklist

## PR Reviewer Checklist
